### PR TITLE
Fix wording of welcome text

### PR DESCRIPTION
**Pull request by fredrivett**
#### Description of what you did:

The wording here is important as it's one of the first things the user
sees of the Strapi project having set it up, and it didn't quite make
sense (though everyone will understand what's meant to be said).

Tweaking this will improve the initial user experience of Strapi.

*Ready to be merged*.

#### My PR is a:

- [ ] 💥 Breaking change
- [ ] 🐛 Bug fix
- [x] 💅 Enhancement
- [ ] 🚀 New feature

#### Main update on the:

- [x] Admin
- [ ] Documentation
- [ ] Framework
- [ ] Plugin

#### Manual testing done on the following databases:

- [x] Not applicable
- [ ] MongoDB
- [ ] MySQL
- [ ] Postgres
- [ ] SQLite

### CODE REVIEW

File: packages/strapi-admin/admin/src/translations/en.json [hunk 1]
```diff
@@ -41,7 +41,7 @@
   "app.components.HomePage.support.link": "GET YOUR T-SHIRT NOW",
   "app.components.HomePage.welcome": "Welcome on board!",
   "app.components.HomePage.welcome.again": "Welcome ",
-  "app.components.HomePage.welcomeBlock.content": "We are happy to have you as one of community member. We are constantly looking for feedback so feel free to send us DM on ",
+  "app.components.HomePage.welcomeBlock.content": "We are happy to have you as one of the community. We are constantly looking for feedback so feel free to send us DM on ",
```
Comment (alexandrebodin) [inline packages/strapi-admin/admin/src/translations/en.json]: I would say "as one of the community members" but not sure about that. What do you think @derrickmehaffy ?

Comment (derrickmehaffy) [reply packages/strapi-admin/admin/src/translations/en.json]: Agreed with your suggestion @alexandrebodin 

`We are happy to have you as one of community members.`

Comment (derrickmehaffy) [reply packages/strapi-admin/admin/src/translations/en.json]: Alternative would be `We are happy to have you as part of the community`

Comment (fredrivett) [reply packages/strapi-admin/admin/src/translations/en.json]: Agree with @derrickmehaffy here, have updated the PR with that copy.

**Pull request merged**